scripts/use-yarn-please.js

```diff
@@ -60,6 +60,12 @@ function checkRepositoryLocation() {
 }
 
 function detectYarnInstallation() {
-  const yarnResult = spawnSync('yarn', ['--version']);
+  let yarnResult;
+  if (process.platform !== 'win32') {
+    yarnResult = spawnSync('yarn.cmd', ['--version']);
+  } else {
```

Comment:
Shouldn't this be if it _is_ win32?

Reply:
Not entirely sure. This file works for me as it is now, maybe I asked you to try it out the first time around? I just copied this from https://github.com/microsoft/fluentui-react-native/pull/2054

Reply:
`.cmd` is Windows only. I think the check was inverted.

Reply:
Got it, fixed